discord/http.py

```diff
@@ -1979,6 +1979,9 @@ def bulk_edit_guild_application_command_permissions(
 
     # Misc
 
+    def get_application(self, application_id: Snowflake, /) -> Response[appinfo.PartialAppInfo]:
+        return self.request(Route("GET", "/applications/{application_id}/rpc", application_id=application_id))
```

Comment:
```suggestion
        return self.request(Route('GET', '/applications/{application_id}/rpc', application_id=application_id))
```

Why is this using the /rpc endpoint? Where is this endpoint documented?

Reply:
Discord does not want to document it for some reason. According to [advaith in the discord-developers server](https://discord.com/channels/613425648685547541/697489244649816084/950540148033081354).

Reply:
I don't think I want to add an undocumented endpoint.

Reply:
FWIW this endpoint has existed for many years and doesn't even need authentication.

Reply:
I'm sure it has, same with the activities endpoint and other undocumented ones. It's just these have no guarantee attached to them so if they break they stick with the library which has stronger versioning guarantees than Discord (e.g. look how long we had support for user accounts) and that's not good.

Reply:
I agree that there's no guarantee. However, something like this is extremely useful and its better to have it with the small possibility it breaks than to not have it at all (IMO).